classifier/StreetPrefixClassifier.js

```diff
@@ -5,11 +5,15 @@ const libpostal = require('../resources/libpostal/libpostal')
 // dictionaries sourced from the libpostal project
 // see: https://github.com/openvenues/libpostal
 
+// prefix languages
+// languages which use a street prefix instead of a suffix
+const prefix = ['fr', 'ca', 'es']
```

Comment:
We can export this list and require it in `StreetSuffixClassifier`

Reply:
Somewhat related, I assume there will have to be a list of countries/languages where the housenumber comes before or after the street name. We already have one in API which is not complete, maybe we can combine the two somehow?

Reply:
We have a client in Catalonia so they'll probably be more coming soon for the Catalonia region & Catalan language :)

Reply:
Yea definitely better to have it in one place, it could live under the `resources` directory if we can't find anywhere else appropriate for it.